Doc/library/cmath.rst

```diff
@@ -85,8 +85,7 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return *e* raised to the power *x*, where *e* is the base of natural
-   logarithms.
+   Return the exponential function at *x*.
```

Comment:
I don't think this is an improvement. "exponential function" is ambiguous - the function `10**x` is an exponential function, for example. (Or at least, many US pre-calculus texts describe it as such.)

We could follow the C standard's lead and use the wording "the base-e exponential function".

Or we could leave as-is. I don't think the existing wording is likely to confuse anyone.

Reply:
> "exponential function" is ambiguous - the function 10**x is an exponential function, for example.

Hmm, I wasn't aware.   Perhaps, I've checked too few sources...
https://mathworld.wolfram.com/ExponentialFunction.html
https://en.wikipedia.org/wiki/Exponential_function
A&S
encyclopediaofmath
maple

Less common seems to be "the exponential of" (Mathematica, numpy, matlab).

> Or we could leave as-is. I don't think the existing wording is likely to confuse anyone.

You can decide:)  Here it is: #101594.  If you think that this will not confuse readers - then it's ok to be as-is.